### PR TITLE
Spanish translations and additions

### PR DESCRIPTION
**Pull request by AcuarioCat**
More Spanish

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -15796,6 +15796,16 @@ fr:jus concentré d'os et de viande de bœuf, jus concentré d'os et de viande d
 # SAUSAGES
 
 # <en:compound
+
+<en:meat
+en:botifarra
+es:butifarra, butifarras
+wikipedia:en:https://en.wikipedia.org/wiki/Botifarra
```
Comment (VaiTon) [inline taxonomies/ingredients.txt:15803]:
```suggestion
wikipedia:en:https://en.wikipedia.org/wiki/Botifarra
wikidata:en:Q7046
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -15796,6 +15796,16 @@ fr:jus concentré d'os et de viande de bœuf, jus concentré d'os et de viande d
...
+
+<en:meat
+es:chistorra
+wikipedia:en:https://en.wikipedia.org/wiki/Chistorra
```
Comment (VaiTon) [inline taxonomies/ingredients.txt:15807]: wikidata:en:Q590686

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:15803]: ok

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:15807]: ok

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -23739,7 +23739,7 @@ fr:graines de millet
 <en:cereal
 en:amaranth
 de:Amarant
-es:amaranto
```
Comment (aleene) [inline taxonomies/ingredients.txt]: inflado can be a process.

File: taxonomies/ingredients.txt [hunk 4]
```diff
@@ -24566,6 +24566,7 @@ nl:bonenmeel
 <en:flour
 <en:flax seed
 en:flax floor, flaxseed floor
```
Comment (aleene) [inline taxonomies/ingredients.txt:24568]: flour?

File: taxonomies/ingredients.txt [hunk 5]
```diff
@@ -46917,6 +46922,8 @@ nl:Gepelde walnoot
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:cerneaux-de-noix
 # 78 products in 4 languages @2018-10-05
 
+<en:wallnut
```
Comment (aleene) [inline taxonomies/ingredients.txt:46925]: wallnut with 2 l's is a typo

File: taxonomies/ingredients_processing.txt [hunk 6]
```diff
@@ -222,12 +222,12 @@ de:ungeschält, ungeschälte
...
 #en:description:To remove the outer covering or shell of something
 en:shelled
-es:sin cascara
+es:sin cascara, pelada, pelado, peladas, pelados
 fr:décortiqué, décortiquée, décortiqués, décortiquées, décoquillé, décoquillée, décoquillés, décoquillées
 
 # en:description:With the outermost layer or skin removed
 en:peeled
-es:sin piel
+es:sin piel, repelada, repelado, repeladas, repelados, descascarrilado, descascarrilada, descascarrilados, descascarriladas
```
Comment (aleene) [inline taxonomies/ingredients_processing.txt:230]: Be careful not to empty a dictionary here. It should really appear on in ingredients. We want to limit the processing power needed.

File: taxonomies/ingredients.txt (hunk 3 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt]: ok, moved to process

File: taxonomies/ingredients.txt (hunk 4 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:24568]: Not mine..but corrected as obviously wrong

File: taxonomies/ingredients.txt (hunk 5 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:46925]: It's everywhere! You're right I'll correct it, obviously been there forever.

File: taxonomies/ingredients_processing.txt (hunk 6 above)
Comment (AcuarioCat) [reply taxonomies/ingredients_processing.txt:230]: I thought the idea was split ingredients from processing? These are all processes. Unfortunately with spanish there are 2 genders and singular/plural so pretty much everything has 4 possible states. I have examples of these processes (at least a single occurance of each) which is why I entered it.

**Comment by aleene** [comment]
Remove your label edit and submit in a different pull request. Maybe it is a chicken-egg issue?

**Comment by stephanegigandet** [comment]
> I do not think that we need to add all variants of a process anymore. @stephanegigandet ?

@aleene, @AcuarioCat : for the ingredients processing, we do need all variations explicitely listed (plurals, feminine etc.) 

e.g. concentrado, concentrada, concentrados, concentradas

There are no automatic plurals like we have for ingredients, in order to minimize the risks of false positives.

File: taxonomies/ingredients_processing.txt [hunk 7]
```diff
@@ -63,14 +63,14 @@ nl:gesneden
 
 # <en:cut
 en:thick cut
-es:cotrada gruesa,cortado grueso
+es:cotrada gruesa,cortado grueso, cortados gruesos,cortadas gruesas
```
Comment (stephanegigandet) [inline taxonomies/ingredients_processing.txt:66]:
```suggestion
es:cortada gruesa,cortado grueso, cortados gruesos,cortadas gruesas 
```

Comment (AcuarioCat) [reply taxonomies/ingredients_processing.txt]: Well spotted!

**Pull request merged**